content/en/blog/2024/inpod-traffic-redirection-ambient/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "Design Istio Ambient to Work with Any Platforms and CNIs"
+description: An innovative traffic redirection mechanism between workload pods and ztunnel.
+publishdate: 2024-01-26
+attribution: "Ben Leggett (Solo.io), Yuval Kohavi (Solo.io), Lin Sun (Solo.io)"
+keywords: [Ambient,Istio,CNI,ztunnel,traffic]
```

Comment:
We seem to be avoiding talking about the Waypoint proxy in this blog post. Is this intentional?

Reply:
Not so much avoiding it, just not introducing it, as Waypoints aren't involved in the problems we had or how we solved them - this is about how to get basic TCP traffic to and from the ztunnel - the waypoint isn't in the picture yet.

Reply:
yep Ben is right, we are not changing waypoint here so no need to mention it. Mark this as resolved.